### PR TITLE
Charter extension 2024-03

### PR DESCRIPTION
**Pull request by pchampin**
[Preview](https://htmlpreview.github.io/?https://raw.githubusercontent.com/w3c/rdf-star-wg-charter/charter-extension-2024-03/index.html) | [Diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Frdf-star-wg-charter%2F&doc2=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Frdf-star-wg-charter%2Fcharter-extension-2024-03%2Findex.html)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star-wg-charter/pull/57.html" title="Last updated on Jun 6, 2024, 2:45 PM UTC (1b504b3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star-wg-charter/57/9ba7884...1b504b3.html" title="Last updated on Jun 6, 2024, 2:45 PM UTC (1b504b3)">Diff</a>

### CODE REVIEW

**Review by rdfguy — APPROVED** [review]
OK as far as I can tell.

File: index.html [hunk 1]
```diff
@@ -140,7 +140,8 @@ <h1 id="title">RDF-star Working Group Charter</h1>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
-        <p>The scope of this Working Group is to extend the recommendations defining RDF 1.1 and SPARQL 1.1 with the features introduced by RDF-star. More precisely, RDF-star introduces the notion of <em>quoted triple</em> to express statements about statements. The abstract and concrete syntaxes of RDF and SPARQL are extended to support this new feature, as well as their respective semantics.</p>
+        <p>The scope of this Working Group is to update the recommendations defining RDF 1.1 and SPARQL 1.1, extending them with the features introduced by RDF-star. More precisely, RDF-star introduces the notion of <em>quoted triple</em> to express statements about statements. The abstract and concrete syntaxes of RDF and SPARQL are extended to support this new feature, as well as their respective semantics.
```
Comment (TallTed) [inline index.html:143]:
```suggestion
        <p>The scope of this Working Group is to update the recommendations defining RDF 1.1 and SPARQL 1.1, extending them with the features introduced by RDF-star. More precisely, RDF-star introduces the notion of <em>quoted triple</em> to express statements about statements. The abstract and concrete syntaxes of RDF and SPARQL, as well as their respective semantics, are to be extended to support this new feature.
```

File: index.html [hunk 2]
```diff
@@ -623,7 +625,7 @@ <h3>
                   28 August 2025
                 </td>
                 <td>
-                  Timeline updated
+                  Rechartered for two more years, to 1) complete the work on RDF 1.2 and SPARQL 1.2 deliverables, and 2) switch to maintenance mode afterwards.
```
Comment (TallTed) [inline index.html:628]:
```suggestion
                  Rechartered for two more years, to 1) complete the work on RDF 1.2 and SPARQL 1.2 deliverables, and 2) switch to maintenance mode thereafter.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by gkellogg** [comment]
The SPARQL 1.2 Community Group is now the SPARQL Dev Community Group.

Even though the RDF Dataset Canonicalization WG charter expires this summer, they should probably be listed for coordination.

**Comment by pchampin** [comment]
> Is this an opportunity to change the name of the WG to "RDF 1.2 Working Group"?

To be clear: in retrospect, I prefer "RDF 1.2", I wish we had chose that name of the WG instead of "RDF-star".
I don't know if it is OK to rename a WG in a rechartering, I'm guessing it would be).

However, I'm afraid that some people might perceive a renaming as an attempt to change the scope of the WG, and that it would create unnecessary friction.

When we have our RECs published (RDF 1.2 xyz and SPARQL 1.2 xyz) and we go to a maintenance group on these specs, I think it will be less controversial to rename to "RDF 1.2 WG". But before that, I'm reluctant.

**Comment by afs** [comment]
> some people might perceive

It's tricky. At the move to a maintenance group, a change of name may cause increased expectations.

How about adding to the description to explain the change of name because of the existing remit over all the docs?

**Comment by gkellogg** [comment]
I wonder if the issue with TAG discussions on Polyglot formats bears any relation on RDF/XML or SPARQL Result formats. It would also impact RDFa and CSVW, which we're not working on, but are related. See https://github.com/ietf-wg-mediaman/suffixes/issues and https://mailarchive.ietf.org/arch/msg/media-types/iWc8TLcWOyO0jyqeiuF9VCZClIs/.

File: index.html [hunk 3]
```diff
@@ -284,7 +296,7 @@ <h4>SPARQL family of specifications</h4>
               <p class="draft-status"><b>Draft state:</b> Adopted from the <a href="https://www.w3.org/2009/sparql/wiki/Main_Page">SPARQL (1.1) WG</a></p>
               <p><b>Adopted Draft:</b> <a href="https://www.w3.org/TR/2013/REC-sparql11-query-20130321/">SPARQL 1.1 Query Language</a>, eds. Steve Harris, Andy Seaborne, W3C Recommendation.</p>
               <p class="draft-status"><b>Reference:</b> <a href="https://www.w3.org/2021/12/rdf-star.html#sparql-star">Section 4 of RDF-star and SPARQL-star</a>, eds. Olaf Hartig, Pierre-Antoine Champin, Gregg Kellogg, Andy Seaborne, W3C Community Group Final Report, 2021. </p>
-              <p class="milestone"><b>Expected completion:</b> 29 February 2024</p>
+              <p class="milestone"><b>Expected completion:</b> Q1 2025</p>
```
Comment (afs) [inline index.html:299]: Here and sparql-results-json, it's Q1 2025, but other SPARQl specs are Q2 and the timeline says "Q2 2025: Recommendation of SPARQL 1.2 deliverables".

```suggestion
              <p class="milestone"><b>Expected completion:</b> Q2 2025</p>
```

File: index.html [hunk 4]
```diff
@@ -304,7 +316,7 @@ <h4>SPARQL family of specifications</h4>
               <p class="draft-status"><b>Draft state:</b> Adopted from the <a href="https://www.w3.org/2009/sparql/wiki/Main_Page">SPARQL (1.1) WG</a></p>
               <p><b>Adopted Draft:</b> <a href="https://www.w3.org/TR/2013/REC-sparql11-results-json-20130321/">SPARQL 1.1 Query Results JSON Format</a>, ed. Andy Seaborne, W3C Recommendation.</p>
               <p class="draft-status"><b>Reference:</b> <a href="https://www.w3.org/2021/12/rdf-star.html#sparql-star-query-results-json-format">Section 4.7.1 of RDF-star and SPARQL-star</a>, eds. Olaf Hartig, Pierre-Antoine Champin, Gregg Kellogg, Andy Seaborne, W3C Community Group Final Report, 2021. </p>
-              <p class="milestone"><b>Expected completion:</b> 29 February 2024</p>
+              <p class="milestone"><b>Expected completion:</b> Q1 2025</p>
```
Comment (afs) [inline index.html:319]:
```suggestion
              <p class="milestone"><b>Expected completion:</b> Q2 2025</p>
```

**Review by TallTed — APPROVED** [review]

**Pull request merged**